libs/dal/src/repositories/job/job.repository.ts

```diff
@@ -215,7 +216,7 @@ export class JobRepository extends BaseRepository<JobDBModel, JobEntity, Enforce
         _subscriberId: this.convertStringToObjectId(job._subscriberId),
         ...(digestKey && { [`payload.${digestKey}`]: digestValue }),
       },
-      '_id'
+      '_id _notificationId'
```

Comment:
We removed the digest filter steps, and in order to replace it's functionality of updating the notificationId after the active digest, we return it here to later update the entity.